README.md

```
@@ -120,20 +100,7 @@ Both deployment methods provide access to:
-
-To switch between deployment methods:
-
-1. Stop current deployment
-2. Navigate to the other deployment folder
-3. Run the setup playbook
-
-Both methods use the same data directories and configuration. 
+- [Admin Security Guide](docs/ADMIN_SECURITY.md) - **Important security information for admin users**
+- [Technical Specification](docs/core/SMS_Bridge_tech_spec_v2.2.md)
+- [Monitoring Specification](docs/core/SMS_Bridge_monitoring_spec_v2.2.md)
+- [Coolify Deployment](coolify/README.md) 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)